markdown/blog/beyond-boundaries.md

```diff
@@ -72,7 +72,7 @@ Comparing this to the previous year, we are proud of this current trajectory and
 
 | No | Project Idea | Mentor/s | Category | Mentee/s |
 | --- | --- | --- | --- | --- |
-| 1 | [Add help/{command} endpoint](https://github.com/asyncapi/server-api/issues/144) | [David Pereira](https://github.com/BOLT04) | Engineering | [Prince Rajpoot](https://github.com/princerajpoot20) |
+| 1 | [Add help/\{command} endpoint](https://github.com/asyncapi/server-api/issues/144) | [David Pereira](https://github.com/BOLT04) | Engineering | [Prince Rajpoot](https://github.com/princerajpoot20) |
```

Comment:
We need to catch more of these errors. I am sure we have them all over breaking the content on website.

Reply:
Any automated solution for this?